### ISSUE TITLE
Provide simple conversions between Int64 and BigInt.

### ISSUE THREAD

**Issue by rakudrama** [dart-lang/fixnum#36]
There is no easy way to convert between Int64 and BigInt, other than via a String.

Example:
```
BigInt value = ...
Int64 i64 = new Int64.fromBigInt(value);
BigInt b = i64.toBigInt();
assert(b == value.toSigned(64));
```
Maybe Int64.fromBigInt should throw on out-ofrange values, since the caller can truncate with

```
new Int64.fromBigInt(value.toSigned(64));
```

**Comment by matanlurey** [comment]
Do we need `Int64` anymore? What value does it have over `BigInt`?

**Comment by rakudrama** [comment]
@matanlurey
Of course, we should migrate protobufs to use BigInt for [u]int64 fields.
The problem is that, today, BigInt performance is not adequate.
Until it is, we should provide efficient and easy to use conversions.

My initial benchmarking indicates that all supported browsers would need to have a native BigInt before we can switch.
The current BigInt implementation is 10x slower than Int64 at conversions to/from String. This could be improved, but I doubt by more than 3x, which leaves a deficit of 3x.

Hopefully browsers are quick to adopt BigInt.

**Comment by matanlurey** [comment]
Ah, that is really good to know.

Maybe on later versions of V8 Int64 should just delegate to (native) BigInt?
